recorded_future/plugin.spec.yaml

```
@@ -27,6 +27,54 @@ hub_tags:
 enable_cache: false
 language: python
 types:
+  list:
+    id:
+      title: ID
+      type: string
+      description: ID
+      required: false
+    name:
+      title: Name
+      type: string
+      description: Name
+      required: false
+    type:
+      title: Type
+      type: string
+      description: Type
+      required: false
+  enterpriseLists:
```

```suggestion
  enterpriseList:
```
It's a single list so the definition should be singular

---

@mrinehart-r7 
In action there are value "fields" which we pass to API. In this fields API need "enterpriseLists". When API gets it it return as API output with values. We only return this value ("enterpriseLists"). We can map this name but in our opinion it could be quite confuse for users. They know that in API there is returned output: "enterpriseLists", but in our output it will be returned as "enterpriseList". Every "fields" are in plural in API.